applications/API/tests/test_endpoints.py

```
@@ -0,0 +1,209 @@
+
+
+
+class TestMyInsultsViewSet(BaseTestCase):
+    def test_list_own_insults(self):
+        """Test that users can list all their insults regardless of status"""
+        self.api_client.force_authenticate(user=self.user)
+        url = reverse("my-insults-list")
+        response = self.api_client.get(url)
+
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertEqual(len(response.data), 2)
```

Ruff (0.12.2)

Use a regular `assert` instead of unittest-style `assertEqual`

Replace `assertEqual(...)` with `assert ...`

(PT009)